modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -128,3 +128,76 @@ test('shows outside pages in path analysis', async ({apiHelpers, page}) => {
 
 	await apiHelpers.jsonWebServicesLayout.deleteLayout(String(sitePage.id));
 });
+
+test('shows tracked pages in path analysis', async ({apiHelpers, page}) => {
+	const pageTitle1 = 'My Page 1';
+
+	const sitePage1 = await createSitePage(apiHelpers, pageTitle1);
+
+	const pageTitle2 = 'My Page 2';
+
+	const sitePage2 = await createSitePage(apiHelpers, pageTitle2);
+
+	const channelName = 'My Property - ' + getRandomString();
+
+	await syncAnalyticsCloud(page, channelName);
+
+	await page.waitForTimeout(10000);
+
+	await page.getByText(pageTitle1).first().click();
+
+	await page.getByText(pageTitle2).first().click();
+
+	await page.waitForTimeout(10000);
+
+	await page.goto(faroConfig.environment.baseUrl);
+
+	await page
+		.getByRole('link', {
+			name: 'FARO-DEV-liferay Liferay Demo Enterprise Plan',
+		})
+		.click();
+
+	await page.locator('.channels-menu.button-root').click();
+
+	await page.getByRole('link', {name: channelName}).click();
+
+	await page.getByRole('link', {exact: true, name: 'Pages'}).click();
+
+	await page.getByRole('button', {name: 'Last 30 days'}).click();
+
+	await page.getByRole('menuitem', {name: 'Last 24 hours'}).click();
+
+	await page
+		.getByRole('cell', {name: 'My Page 1 - Liferay DXP'})
+		.getByRole('link')
+		.click();
+
+	await page.getByRole('link', {name: 'Path'}).click();
+
+	await expect(page.getByText('Home - Liferay ...')).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('My Page 2 - Lif...')).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).first()).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).nth(1)).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).nth(2)).toBeVisible({
+		timeout: 100 * 1000,
+	});
```

Comment:
Hey @rachaelkoestartyo, I think these should be the pages you should check to see if they appear in Path Analysis. What do you think?

**Steps:**
1. Go to My Page 1 (Using the `navigateToSitePage` function) - Direct access
2. Go to My Page 2 (Using `page.getByText`) - Exit Page
3. Go to My Page 1 (Using `page.getByText`) - Referral Page
4. Go to AC > Go to Pages > Path Analysis of My Page 1
5. Check that in the Referral column you have direct access and My Page 2 as referral pages
6. Check that in the Exit Page column has Drop Offs and My Page 2 as exit pages